### PR TITLE
Fix Namespace definition

### PR DESCRIPTION
**Pull request by LipkeGu**
This Pr fixes a typo in a namespace definition.

### CODE REVIEW

File: OpenRA.Mods.RA/Scripting/Properties/InfiltrateProperties.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@
 using OpenRA.Mods.RA.Activities;
 using OpenRA.Scripting;
 
-namespace OpenRA.RA.Common.Scripting
+namespace OpenRA.Mods.Common.Scripting
```
Comment (penev92) [inline OpenRA.Mods.RA/Scripting/Properties/InfiltrateProperties.cs]: This should actually be OpenRA.Mods.RA.Scripting to match the folder.

**Comment by penev92** [comment]
:+1:

**Comment by GraionDilach** [comment]
:+1:

**Comment by penev92** [comment]
Thanks.

**Pull request merged**